.github/workflows/release.yml

```
@@ -0,0 +1,139 @@
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - os: ubuntu-latest
+            plat: linux-x64
+          - os: macos-latest
+            plat: macos-arm64
+          - os: windows-latest
+            plat: windows-x64
+    steps:
+      - uses: actions/checkout@v4
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected ',' or '}'

(syntax-check)